pkg/splunk/enterprise/afwscheduler.go

```diff
@@ -700,11 +700,42 @@ func installApp(rctx context.Context, localCtx *localScopePlaybookContext, cr sp
 		return fmt.Errorf("app pkg missing on Pod. app pkg path: %s", appPkgPathOnPod)
 	}
 
+	// gather installed app name
+	// this is the name of top folder when we untar the app archive package
+	if worker.appDeployInfo.InstalledAppName == "" {
```

Comment:
Since we are about to install the app, this name `InstalledAppName` is a misnomer. It should be something like `actualAppName` or something along those lines.

Reply:
Good point Gaurav. I have changed the `InstalledAppName` to `AppPackageTopFolder` to better match where this name is coming from. I have also left some comments for the existing AppName that this is really an app archive name.